buildscripts/test.sh

```diff
@@ -33,6 +33,7 @@ fi
 
 PACKAGES=$(go list ./... | grep -v '/vendor/\|/pkg/apis/\|/pkg/client/\|tests')
 for d in $PACKAGES; do
+    echo "Running for $d"
```

Comment:
```suggestion
        echo "Running for $d"
```

Reply:
I added it for debugging purpose will remove it